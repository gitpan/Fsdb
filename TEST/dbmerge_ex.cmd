prog='dbmerge'
args='--input - --input TEST/dbmerge_ex_b.in cname'
cmp='diff -cb '
