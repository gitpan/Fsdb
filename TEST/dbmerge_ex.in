#h cid cname
11 numanal
10 pascal
