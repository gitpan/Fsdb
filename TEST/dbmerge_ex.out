#fsdb cid cname
11	numanal
12	os
10	pascal
13	statistics
#   | dbmerge --input - --input TEST/dbmerge_ex_b.in cname
